### ISSUE TITLE
Compilation error in Trilinos due to redeclaration in Tpetra CrsMatrix

### ISSUE THREAD

**Issue by ikalash** [trilinos/Trilinos#3339]
The Albany nightlies failed to build Trilinos last night due to an issue in Tpetra:
```
repos/Trilinos/packages/tpetra/core/src/Tpetra_CrsMatrix_decl.hpp:4928:101: error: redeclaration of ‘template Teuchos::RCP Tpetra::importAndFillCompleteCrsMatrix(const Teuchos::RCP&, Tpetra::Import&, const Teuchos::RCP >&, const Teuchos::RCP >&, const Teuchos::RCP&)’ may not have default arguments [-fpermissive]
```
Here is the full error from CDash: http://cdash.sandia.gov/CDash-2-3-0/viewBuildError.php?buildid=75020 .  Can someone please look into this?

@trilinos/tpetra

**Comment by rppawlo** [comment]
@mhoemmen fixed this in #3333 . Fix was pushed early this morning.

**Comment by mhoemmen** [comment]
👍👍👍👍👍👍
